src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/test/VedtakBegrunnelserContext.test.ts

```diff
@@ -44,7 +49,8 @@ describe('VedtakBegrunnelserContext', () => {
                 ];
                 const perioder = filtrerOgSorterPerioderMedBegrunnelseBehov2(
                     vedtaksperioder,
-                    erLesevisning
+                    erLesevisning,
+                    BehandlingResultat.FORTSATT_INNVILGET
```

Comment:
innvilget_og_opphørt? :) Har vel ikke noe å si for testresultatet, men hender jo noen forsøker å forstå domene/funksjon ved å lese tester.